Description: Remove test_fqdn()
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2022-09-14

--- python-ephemeral-port-reserve-1.1.4.orig/tests/ephemeral_port_test.py
+++ python-ephemeral-port-reserve-1.1.4/tests/ephemeral_port_test.py
@@ -44,11 +44,6 @@ def test_localhost():
     assert_ip(LOCALHOST)
 
 
-def test_fqdn():
-    fqip = socket.gethostbyname(getfqdn())
-    assert_ip(fqip)
-
-
 def test_preferred_port():
     import ephemeral_port_reserve
     print(ephemeral_port_reserve)
